pyproject.toml

```diff
@@ -64,6 +64,10 @@ tox-pyenv = "*"
 "conductor" = "maestrowf.conductor:main"
 
 
+
+[tool.poetry.group.dev.dependencies]
+pytest = ">=7.2.0"
```

Comment:
this one need to be specifically 7.2 for any reason vs the more permissive 7.x?

Reply:
Odd -- it's listed as `'*'` on my end.

Reply:
Oh -- nevermind this is in a different group.